website/docs/r/policy_tier0_gateway_gre_tunnel.html.markdown

```diff
@@ -0,0 +1,137 @@
+---
+subcategory: "Gateways and Routing"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_tier0_gateway_gre_tunnel"
+description: A resource to configure a GRE tunnel on Tier-0 gateway on NSX Policy manager.
+---
+
+# nsxt_policy_tier0_gateway_gre_tunnel
+
+This resource provides a method for the management of a Tier-0 gateway GRE tunnel. Note that edge cluster and in interface must be configured on Tier-0 Gateway in order to configure GRE tunnels on it.
+
+This resource is applicable to NSX Policy Manager.
+
+# Example Usage
+
+```hcl
+data "nsxt_policy_edge_cluster" "EC" {
+  display_name = "EDGECLUSTER1"
+}
+
+data "nsxt_policy_transport_zone" "test" {
+  display_name = "tz-vlan"
+}
+resource "nsxt_policy_vlan_segment" "test" {
+
+  transport_zone_path = data.nsxt_policy_transport_zone.test.path
+  display_name        = "interface_test"
+  vlan_ids            = [11]
+  subnet {
+    cidr = "10.2.2.2/24"
+  }
+}
+data "nsxt_policy_edge_node" "EN" {
+  edge_cluster_path = data.nsxt_policy_edge_cluster.EC.path
+  member_index      = 0
+}
+
+resource "nsxt_policy_tier0_gateway" "test" {
+  display_name = "terraform-acctest-resource-74743"
+  ha_mode      = "ACTIVE_STANDBY"
+
+  edge_cluster_path = data.nsxt_policy_edge_cluster.EC.path
+
+}
+
+resource "nsxt_policy_tier0_gateway_interface" "test" {
+  display_name   = "terraform-acctest-resource-869"
+  description    = "Acceptance Test"
+  type           = "EXTERNAL"
+  mtu            = 1500
+  gateway_path   = nsxt_policy_tier0_gateway.test.path
+  segment_path   = nsxt_policy_vlan_segment.test.path
+  edge_node_path = data.nsxt_policy_edge_node.EN.path
+  subnets        = ["192.168.240.10/24"]
+  enable_pim     = "true"
+  urpf_mode      = "STRICT"
+
+
+  tag {
```

Comment:
since this is a resource for dependency,  I think we should provide minimal working config (tags etc. not needed)